utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export const formatBalance = (balance) =>
+	balance - Math.round(balance) !== 0 ? balance.toFixed(4) : balance;
```

Comment:
as per our Slack convo
- [ ] Add parameter type and return type so it's clear what's the function is doing
- [ ] Add logic to check for balance that is less than 4 dp, in that case output straight as `< 0.0001`
- [ ] Need some unit testing for this function to cover all cases scenario

I also think it's fine to output the number with .0000 at the end, so you don't need to check if the number is integer or not, so `balance - Math.round(balance) !== 0 ? balance.toFixed(4) : balance;` can just be as `balance.toFixed(4)`